packages/api/models/executions.js

```diff
@@ -52,7 +45,30 @@ class Execution extends Manager {
       createdAt: get(payload, 'cumulus_meta.workflow_start_time'),
       timestamp: Date.now()
     };
+    return doc;
+  }
+
 
+  // Update database row
+  async updateExecutionFromSns(payload) {
```

Comment:
This could be easily DRY-ed up via a method argument, however that would require changing the signature of createExecutionsFromSns and I'm uncertain about external contracts and how much that detracts given the relative lack of complexity.